data/campaigns/Northern_Rebirth/scenarios/13a_Showdown.cfg

```diff
@@ -47,11 +47,13 @@
         canrecruit=yes
         side=1
         controller=human
-        recruit=Thug,Poacher,Spearman,Bowman,Dwarvish Fighter,Dwarvish Thunderer,Dwarvish Ulfserker,Dwarvish Scout,Gryphon,Footpad
+        recruit=Thug,Poacher,Spearman,Bowman,Dwarvish Fighter,Dwarvish Thunderer,Dwarvish Ulfserker,Dwarvish Scout,Gryphon,Footpad,Fencer,Heavy Infantryman
         team_name=knalgans
         user_team_name=_"Alliance"
-        gold=100
+        {GOLD 900 500 100}
```

Comment:
@Toranks is this too much?

Reply:
I don't really understand how the gold is distributed in NR yet. I've seen on the forum people with 2000 gold at the start of this scenario on nightmare, I had around 600 with Tallin and it's costing me a lot. I haven't finished the scenario yet.

Reply:
Did you manage to retrieve the stolen gold that originally came from The Pursuit's treasury?

Reply:
Yes, I retrieve the gold but is calculated and divided in a strange way

Reply:
Comments in the WML would help, saying why these numbers were chosen.

Reply:
> Comments in the WML would help, saying why these numbers were chosen.

I think it was however much was necessary for me to recall my entire recall list on EASY, and then the NORMAL amount was an interpolation of the halfway point between that and the existing HARD amount, but I can't remember for certain...

Reply:
Comment added in [743a71c](https://github.com/wesnoth/wesnoth/pull/6057/commits/743a71cb176982fd79d0a2ace9f5c8111f369ad4).